postgresql/config/functions.sh

```diff
@@ -59,13 +58,12 @@ master_ready() {
 bootstrap_replica_via_pg_basebackup() {
   echo 'Bootstrapping replica via pg_basebackup from leader '
 
-  rm -rf {{pkg.svc_data_path}}/*
-  pg_basebackup --pgdata={{pkg.svc_data_path}} --xlog-method=stream --dbname='postgres://{{cfg.replication.name}}@{{svc.leader.sys.ip}}:{{cfg.port}}/postgres'
+  rm -rf {{cfg.data_directory}}/*
```

Comment:
Note:  we should put some kind of guard around this,  in case `{{cfg.data_directory}}` evaluates to "" and we blow away `/`